launcher/src/background.js

```diff
@@ -1023,30 +1019,62 @@ app.on("web-contents-created", (event, contents) => {
   });
 });
 
-// This method will be called when Electron has finished
-// initialization and is ready to create browser windows.
-// Some APIs can only be used after this event occurs.
 app.on("ready", async () => {
-  // workaround for linux whitescreen
-  // if(process.platform === "linux"){
-  //   app.commandLine.appendSwitch('--no-sandbox')
-  // }
+  if (process.env.WEBPACK_DEV_SERVER_URL) {
+    // Development mode
+    app.setAsDefaultProtocolClient("stereumlauncher", process.execPath, [path.resolve(process.argv[1])]);
+  } else {
+    // Production mode
+    app.setAsDefaultProtocolClient("stereumlauncher");
+  }
+  await createWindow();
+});
 
-  // Disable "View" and "Window" Menu items in build (since CTRL+R and F5 is disabled also)
-  if (!isDevelopment) {
-    const hideMenuItems = ["viewmenu", "windowmenu"];
-    var menu = Menu.getApplicationMenu();
-    menu.items.filter((item) => hideMenuItems.includes(item.role)).map((item) => (item.visible = false));
-    Menu.setApplicationMenu(menu);
-    stereumUpdater.checkForUpdates();
```

Comment:
i think you remeoved `stereumUpdater.checkForUpdates()`

Reply:
seems, u r right.